docs/transactions.md

```diff
@@ -827,13 +827,14 @@ will be in a pending state until sufficient number of approvals is received.
 
 The certificate is immutable. It can only be revoked by either the owner or a quorum of Trustees.
 
```

Comment:
What if we rename the commands as follows:
# X509 PKI
## Device Attestation Certificates (DA)
### PROPOSE_ADD_PAA
### APPROVE_ADD_PAA
### REJECT_ADD_PAA
### PROPOSE_REVOKE_PAA
### APPROVE_REVOKE_PAA
### ASSIGN_VID_TO_PAA
### ADD_REVOCATION_DISTRIBUTION_POINT
### UPDATE_REVOCATION_DISTRIBUTION_POINT
### DELETE_REVOCATION_DISTRIBUTION_POINT
### ADD_PAI
### REVOKE_PAI
### REMOVE_PAI
### GET_REVOCATION_DISTRIBUTION_POINT
### GET_REVOCATION_DISTRIBUTION_POINT_BY_SKID
### GET_ALL_REVOCATION_DISTRIBUTION_POINT
### GET_PROPOSED_PAA
### GET_REJECTED_PAA
### GET_PROPOSED_REVOKE_PAA
### GET_ALL_PAA
### GET_ALL_REVOKED_PAA
### GET_ALL_PROPOSED_PAA
### GET_ALL_REJECTED_PAA
### GET_ALL_PROPOSED_REVOKE_PAA
## E2E (NOC/ICA)
### ADD_NOC
### REVOKE_NOC
### ADD_ICA
### GET_NOC_BY_VID
### GET_ICA_BY_VID
### GET_REVOKED_NOC
### GET_ALL_NOC
### GET_ALL_ICA
### GET_ALL_REVOKED_NOC
## Common
### GET_CERT
### GET_REVOKED_CERT
### GET_CERTS_BY_SKID
### GET_CERTS_BY_SUBJECT
### GET_CHILD_CERTS
### GET_ALL_CERTS
### GET_ALL_REVOKED_CERTS

Reply:
Renamed